app/src/docs/manageAccess.md

```diff
@@ -0,0 +1,156 @@
+# Access Management Guide
+
+## Overview
+
+Packit uses a role-based access control (RBAC) system that provides flexible, fine-grained control over user permissions. The system supports both role-based permissions and direct user permissions, with the ability to apply permissions globally or scope them to specific resources.
+
+## Core Concepts
+
+### Users and Roles
+
+- **Roles** contain permissions that are automatically inherited by all assigned users
+- **Users** can have direct permissions in addition to those inherited from their roles
+- **Permission Inheritance** combines all role permissions with direct user permissions, with the most permissive scope taking precedence
+
+### Permission Resolution
+
+When determining a user's effective permissions, the system evaluates:
+
+1. **Role permissions** - All permissions from assigned roles
+2. **Direct user permissions** - Specific permissions assigned directly to the user
+3. **Scope precedence** - Global permissions override scoped permissions for the same action
+
+## Permission Types
+
+### Global Permissions
+
+These permissions apply across the entire application and cannot be scoped to specific resources.
+
+| Permission      | Description                    | Capabilities                                                                                                                                                            |
+| --------------- | ------------------------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `user.manage`   | Complete administrative access | • Manage users and roles<br>• Assign permissions to users and roles<br>• Manage read permissions on packets and packet groups<br>• Automatic read access to all packets |
+| `packet.run`    | Execute packet groups          | • Access packet runner interface<br>• Run any packet groups<br>• View packet run logs                                                                                   |
+| `outpack.read`  | Read outpack server data       | • Make read calls to the outpack server                                                                                                                                 |
+| `outpack.write` | Write outpack server data      | • Make write calls to the outpack server<br>• Includes `outpack.read` capabilities                                                                                      |
```

Comment:
This permission could be rewritten as `outpack.push` but it means that the user can push packets directly into the server. It requires `outpack.read` which is why you've had to include it.